io/www/src/fragment/pipeline.js

```diff
@@ -26,6 +26,7 @@ const RESPONSE_HEADERS = {
     'Access-Control-Expose-Headers': 'X-Request-Id,Etag,Last-Modified,server-timing',
     'Content-Type': 'application/json',
     'Content-Encoding': 'br',
+    'Cache-Control': 'public, max-age=300, stale-while-revalidate=86400',
```

Comment:
can you add 300 & 86400 as default values, and request for maxage & stale values from networkConfig?